db/get.go

```diff
@@ -5,18 +5,18 @@ import (
 	"github.com/jmoiron/sqlx"
 )
 
-func Get(db sqlx.Queryer, dest interface{}, qry squirrel.SelectBuilder) error {
+func Get(db sqlx.Queryer, destVar interface{}, qry squirrel.SelectBuilder) error {
```

Comment:
Naming does not correlate to possible functionality. Please revert.

Reply:
I think this name is better because it more clearly shows what is needed. The function may expect an interface, but the underlying code will panic if you pass in a slice. Similarly the Select function will panic if you don't pass a slice. Having these as parameter names makes it clearer which one you should use.

Reply:
Ah, right. I'd advise using "value"/"v" and "slice". As to keep as straightforward and simple as possible.

Reply:
Yeah val is better.